src/routers/SearchRouter.js

```diff
@@ -32,7 +34,15 @@ class SearchRouter extends ApplicationRouter<Array<Object>> {
     const cachedValue = cache.get(query);
 
     const allStops = await AllStopUtils.fetchAllStops();
-    const formattedStops = allStops.map(s => ({
+    const filteredStops = allStops.filter(s => (
+      fuzz.partial_ratio(s.name.toLowerCase(), query.toLowerCase()) >= MIN_FUZZ_RATIO
+    ));
+    filteredStops.sort((a, b) => {
+      const aFuzzRatio = fuzz.partial_ratio(query, a.name.toLowerCase());
+      const bFuzzRatio = fuzz.partial_ratio(query, b.name.toLowerCase());
```

Comment:
query is lowercased up above but not here, is there any reason? Also, if it's supposed to be lowercased, can we create a variable for it instead? maybe `normalizedQuery`

Reply:
@kevinchan159 why did you resolve this conversation 😠, please enlighten me